docs/hive-integration/crds/agentClusterInstall-dualstack-SNO.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: extensions.hive.openshift.io/v1beta1
+kind: AgentClusterInstall
+metadata:
+  name: test-agent-cluster-install
+  namespace: spoke-cluster
+spec:
+  clusterDeploymentRef:
+    name: single-node-dual-stack
```

Comment:
nit: maybe just use 'single-node' here , since this is what we have in CRD examples? 
https://github.com/openshift/assisted-service/blob/577382f491b06a18c61c04076c1eba6c2e71cb4d/docs/hive-integration/crds/clusterDeployment.yaml#L4

Reply:
Yes, makes very much sense. Fixed